configure.sh

```diff
@@ -63,4 +74,42 @@ cd /home/vagrant/torque-sites/base/simplesaml/ansible
 envsubst < inv/local/group_vars/all.tmpl > inv/local/group_vars/all
 ansible-playbook simplesaml.yml -i inv/local
 
+# Set up ETL
+if [ -z "$OTS_SVN_USERNAME" ]
+then
+	echo "Skipping ETL Setup (OTS_SVN_USERNAME not set)"
+	ETL_ENABLED=false
+else
+	echo "Set up ETL"
+	export OTS_DIR=/home/vagrant/data/ots
+	mkdir -p $OTS_DIR/clients/lever-for-change/torque-sites
+	mkdir -p $OTS_DIR/utils
+
+	# Save SVN Credentials
+	svn list \
```

Comment:
If the point of this is just to get the creds saved, redirect output to `/dev/null` so that it's obvious what's going on.

Reply:
Good thought -- will do